### PR TITLE
add support for fractional seconds

### PR DESCRIPTION
**Pull request by ziv**
strptime (the opposite of strfftime) support microseconds (%f) or in momentjs jargon "Fractional Second".
The flag added to the "replacement" list

### CODE REVIEW

**Comment by ziv** [comment]
reference: http://pubs.opengroup.org/onlinepubs/009695399/functions/strptime.html

File: spec/strftime.spec.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ describe('strftime', function () {
   beforeEach(function () {
     january17 = moment('2012-01-17T19:54:20');      // Tuesday
     december2 = moment.utc('2029-12-02T01:02:03Z'); // Sunday
+    farction = moment('2012-01-17T19:54:20.123');   // Tuesday with microseconds
```
Comment (benjaminoakes) [inline spec/strftime.spec.js]: farction -> fraction

File: spec/strftime.spec.js [hunk 2]
```diff
@@ -153,6 +154,12 @@ describe('strftime', function () {
     });
   });
 
+  describe('given %f', function () {
+      it('gives the micro-seconds', function () {
+          expect(farction.strftime('%f')).toEqual('123');
+      });
```
Comment (benjaminoakes) [inline spec/strftime.spec.js]: Two spaces please.

**Comment by benjaminoakes** [comment]
Released as v0.2.3

**Pull request merged**